frontend/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "HangHub",
-  "version": "1.0.1",
-  "description": "HangHub is a productivity tool that allows you to see users who are working on the same GitHub issue as you. It is available as a browser extension for Chrome and Firefox.",
-  "main": "src/index.js",
-  "scripts": {
-    "build": "webpack --mode=production",
-    "lint": "eslint ./src"
-  },
+  "name": "hang-hub-frontend",
+  "version": "0.0.1",
+  "private": true,
+  "description": "",
```

Comment:
Why did you remove the description?

Reply:
Ah, I see you moved it. Still, maybe we should provide some, simple describtion for backend and the fronend part of this plugin. Explain that the frontend part if the browser extension for Chrome and FF and the backend part support the comunication between clients and does not store any users data pernamently.

Reply:
Do we need this description everywhere? We don't' publish this package on NPM so this description won't be used anywhere.